seattle_example/seattle_tasks.yml

```diff
@@ -33,7 +33,9 @@
 #        - { src: maryland_blkgrps, dest: '' }
 
 - name: Copy geo data to server
-  copy: src=seattle_example/{{ item.src }} dest=/opt/piecewise_web/{{ item.dest }}
+  copy:
+    src: seattle_example/{{ item.src }}
+    dest: /opt/piecewise_web/{{ item.dest }}
   with_items:
     - { src: seattle_census10_blockgroups.topojson, dest: seattle_census10_blockgroups.topojson }
```

Comment:
What about the last item?  - { src: center.js, dest: js/center.js }

Do you think that should get copied in another copy operation?